### ISSUE TITLE
BrowserWindow vibrancy not work in 1.6.0

### ISSUE THREAD

**Issue by ektx** [electron/electron#8714]
* Electron version: 1.6.0
* Operating system: mac 10.12.3

### Expected behavior
Hi everyone, i want to use vibrancy in my app, but it not work! somebody can help me?

```js
const electron = require('electron');
const {app, BrowserWindow} = electron;

function createWindow() {

	let win = new BrowserWindow({
		width: 800,
		height: 600,
		frame: false,
		vibrancy: 'dark'
	})
	
}

app.on('ready', createWindow);

```

**Comment by groundwater** [comment]
Hey @ektx thanks for trying out 1.6.0. Can you confirm that this works as expected in our current stable release `1.4.15`?

**Comment by ektx** [comment]
@groundwater yes,  it can work in 1.4.15 !

**Comment by gerhardberger** [comment]
@groundwater @zeke: I took a look at it and it was just a version check issue. Fixed in #8717

**Comment by kevinsawicki** [comment]
Fixed in #8717
